sky/global_user_state.py

```diff
@@ -159,7 +162,13 @@ def add_or_update_cluster(cluster_name: str,
             cluster_name,
             ClusterStatus.STOPPED.value,
         ))
+
+    if not metadata:
+        metadata = {}
+        metadata['usage_intervals'] = []
```

Comment:
How about we set the starting time here, when there is no starting time set? The `launch_time` get at the `remove_cluster` is not accurate, as it will be updated every time `sky launch` is called on an existing cluster.